apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/MapLikeToContainInOrderOnlyCreatorSamples.kt

```diff
@@ -6,13 +6,41 @@ import kotlin.test.Test
 
 class MapLikeToContainInOrderOnlyCreatorSamples {
 
+    @Test
+    fun entries() {
+        expect(mapOf(1 to "a", 2 to "b")).toContain.inOrder.only.entries(
+            1 to "a", 2 to "b"
+        )
+
+        fails {
+            expect(mapOf(1 to "a", 2 to "b")).toContain.inOrder.only.entries(
+                2 to "b", // fails because subject does not have the same order
+                1 to "a",
+            )
+        }
+    }
+
+    @Test
+    fun entriesKeyValue() {
+        expect(mapOf(1 to "a", 2 to "b")).toContain.inOrder.only.entries(
+            KeyValue(1) { toEqual("a") },
```

Comment:
Are the `KeyValue` usages accurate? Does this convey how they're meant to be used?

Reply:
they are good :+1: